src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+const SearchForm = ({ handleSearchArticles }) => {
+  const [isQuery, setIsQuery] = useState("");
+
+  const handleButtonClick = () => {
+    handleSearchArticles(`${isQuery.toLowerCase()}`);
+  };
+
+  const handleEnterKey = (e) => {
```

Comment:
`onKeyDown={handleEnterKey}` will not be needed here because you need to add `onSubmit` to the form